lib/matplotlib/tests/test_axes.py

```diff
@@ -2958,6 +2958,29 @@ def test_hist_stacked_weighted():
     ax.hist((d1, d2), weights=(w1, w2), histtype="stepfilled", stacked=True)
 
 
+@image_comparison(baseline_images=['stem'], extensions=['png'], style='mpl20',
+                  remove_text=True)
+def test_stem():
```

Comment:
Does anyone know a nice way to merge these two tests together to avoid code duplicaiton?

Reply:
Repeat 'stem' twice in the baseline_image args and just generate two figures in a loop(?)